src/lambdas/breeds-get.ts

```
@@ -0,0 +1,38 @@
+import * as domain from '../domain'
+import { BreedsService } from '../services/breeds'
+import * as Config from './config'
+import { Response } from './types'
+
+namespace BreedsResponse {
+  export interface t extends Response {
+    body: domain.breeds.FlatBreed[]
+  }
+
+  export function make(result: domain.breeds.FlatBreed[]): t {
+    return { statusCode: 200, body: result }
+  }
+}
+
+namespace ErrorResponse {
+  export interface t extends Response {
+    message: string
+  }
+
+  export function make(message: string): t {
+    return { statusCode: 500, message }
+  }
+}
+
+export async function handler(
+  config_: Config.t = Config.envConfig,
+): Promise<BreedsResponse.t | ErrorResponse.t> {
+  try {
+    const breedsService = new BreedsService(config_.breeds)
+    const breeds = await breedsService.listAll()
+    const result = domain.breeds.flatten(breeds)
+    return BreedsResponse.make(result)
+  } catch (err: unknown) {
+    console.log(err)
```

We really want something like `logger.error("Error:", err )`  with structured logging here, but this will have to do for now (at least it doesn't hide the error).